diff --git a/node_modules/keccak256/dist/keccak256.js b/node_modules/keccak256/dist/keccak256.js
index 9ca0b64..1096368 100644
--- a/node_modules/keccak256/dist/keccak256.js
+++ b/node_modules/keccak256/dist/keccak256.js
@@ -3,9 +3,13 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 const bn_js_1 = __importDefault(require("bn.js"));
-const buffer_1 = __importDefault(require("buffer"));
 const keccak_1 = __importDefault(require("keccak"));
-const Buffer = buffer_1.default.Buffer;
+import { Buffer } from 'buffer'
+globalThis.Buffer = Buffer
+
 function keccak256(value) {
     value = toBuffer(value);
     return (0, keccak_1.default)('keccak256').update(value).digest();
